Automattic/jetpack
# CLI: Add install command

#### Changes proposed in this Pull Request:
* Adds `jetpack install` which runs `composer install` and `yarn install` if composer.json or package.json are present in a project.

Also, prettifies the build step:

https://user-images.githubusercontent.com/88897/105559526-0d8cfe00-5cd7-11eb-8ee4-bf819ec4a47e.mov

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
n/a

#### Testing instructions:
* `yarn install` for the initial install.
* Either run `yarn cli-link` or prefix all of the commands below with `.tools/cli/bin/`, e.g. `./tools/cli/bin/jetpack install --help`.
* `jetpack` to see all help commands.
* `jetpack install --help` to see the install commands.
* `jetpack install` to get a project picker.
* `jetpack install --root` to get a project picker and install the monorepo root too.
* `jetpack install monorepo` to install the monorepo root without a project.
* `jetpack install packages/connection` to install that particular package.
* `jetpack install packages/connection --root` to install a particular package and the repo root.
* `jetpack build plugins/jetpack` to see the new build prettification.

#### Proposed changelog entry for your changes:


*


https://user-images.githubusercontent.com/88897/105548973-f7c30d00-5cc5-11eb-82fa-c6a8f9b0e315.mov
